src/main/java/edu/ucsb/cs156/happiercows/controllers/CowDeathController.java

```diff
@@ -0,0 +1,101 @@
+package edu.ucsb.cs156.happiercows.controllers;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.ucsb.cs156.happiercows.entities.CowDeath;
+import edu.ucsb.cs156.happiercows.repositories.CowDeathRepository;
+
+@Slf4j
+@Api(description = "Cow Death")
+@RequestMapping("/api/cowdeath")
+@RestController
+public class CowDeathController extends ApiController {
+    
+    @Autowired
+    private CowDeathRepository cowDeathRepository;
+
+    @Autowired
+    ObjectMapper mapper;
+
+    @ApiOperation(value = "Create a new CowDeath entity")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PostMapping(value = "", produces = "application/json")
+    public ResponseEntity<String> createCowDeath(
+        @ApiParam("commons_id") @RequestParam long commonsId,
+        @ApiParam("user_id") @RequestParam long userId,
+        @ApiParam("zonedDateTime") @RequestParam("zonedDateTime") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime zonedDateTime,
```

Comment:
We will want to remove this parameter before merging (the team doesn't have to do this; the staff can take care of it.)